src/main/java/it/gov/pagopa/payhub/activities/dto/classifications/ClassifyResultDTO.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.payhub.activities.dto.classifications;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class ClassifyResultDTO {
```

Comment:
As discussed on Slack, we could just reason on IUV, thus we could avoid a separate DTO, thus this class could be deleted

Reply:
to discuss